tap_hubspot/hubspot.py

```diff
@@ -18,8 +18,16 @@ class InvalidCredentials(Exception):
     pass
 
 
-def backoff_with_offset(backoff, offset=10):
-    return lambda: (n + offset for n in backoff)
+def giveup_http_codes(e: Exception):
+    if not isinstance(e, requests.RequestException):
+        return False
```

Comment:
Kind of redundant.

Reply:
true..

Reply:
although it means I can not worry about all other things further down, ie. guard clause and inverting all the remaining checks which are subclasses of RequestException